app/controllers/rubcop_test_controller.rb

```diff
@@ -0,0 +1,18 @@
+class RubocopController  < ApplicationController
+	def initialize
+		print "this is so bad patern"
```

Comment:
__[rubocop]__
* Style/Tab: Tab detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Style/IndentationWidth: Use 2 (not 1) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)